core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/configuration/security/jwt/ApiError.java

```diff
@@ -0,0 +1,40 @@
+package io.metadew.iesi.server.rest.configuration.security.jwt;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.http.HttpStatus;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+
+@Getter
+@Setter
+public class ApiError {
+
+    private int status;
+    private LocalDateTime timestamp;
+    private String message;
+
+    public ApiError(HttpStatus status, Throwable ex) {
+        super();
+        this.timestamp = LocalDateTime.now();
+        this.status = status.value();
+        this.message = ex.getMessage();
+    }
+
+    public String convertToJson() throws IOException {
+        try {
+            ObjectMapper mapper = new ObjectMapper();
+            mapper.registerModule(new JavaTimeModule());
+            mapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+            return mapper.writeValueAsString(this);
+        } catch (IOException e) {
+            e.printStackTrace();
+            throw new IOException(e);
```

Comment:
you throw the Exception in the method definition and catch in the method execution. Please delete the capture in the method execution